src/main/java/Entities/Player.java

```diff
@@ -195,88 +212,96 @@ public String riggedRoll(int rig) {
+            } else {
+                this.move(a + b);
+                return (a + "\n" + b);
             }
-            return (a + " " + b + "\n");
         }
+        this.move(a + b);
+        return (a + "\n" + b);
     }
 
     /**
-     * This method allows this player to build houses on a currently owned property
+     * Builds a house on the current property
      * @param property the property to build the house on
      * @param houses the number of houses to build
      */
     public void buildHouse(Property property, int houses) {
         if (properties.contains(property)) {
-            property.addHouse(this, houses);
+            //property.addHouse(houses);
         }
     }
 
     /**
-     * Increases the amount of money owned by this player
-     * @param change an integer indicating how much money will be added to the player's balance
+     * Increases the money of the player
+     * @param change the integer to change the money by
      */
     public void changeMoney(int change) {
-
         money += change;
     }
 
     /**
-     * A payment method where money is deducted from the player's balance e.g. to pay tax
-     * @param money the amount of money deducted from this player's balance
+     * Decrease the money of the player
+     * @param money the amount of money to decrease by
      */
     public void pay(int money) {
-
         this.money -= money;
     }
 
     /**
-     * A payment method that allows this player to pay another player e.g. paying rent
-     * @param player the player whom this player is giving money to
-     * @param money the amount of money being paid by this player
+     * Decrease the money of the current player and add it to another players money
+     * @param player the player to add the money to
+     * @param money the amount of money to decrease by for current player and increase by for the other player
      */
     public void pay(Player player, int money) {
         this.money -= money;
         player.money += money;
     }
 
     /**
-     * This method allows the player to place one of their owned properties for mortgage
-     * @param property the property to mortgaged by this player
+     * Placing the property for mortgage
+     * @param property the property to remove and to add the money to the current players balance
      */
     public void mortgage(Property property) {
         this.properties.remove(property);
         this.money += property.getMortgageValue();
     }
 
+    /**
+     * Gets all of the properties of the current player
+     * @return returns and array list of properties
+     */
+    public ArrayList<Property> getProperties() {
+        return properties;
+    }
+
+    /**
+     * Sets the current players properties to another ArrayList of properties
+     * @param properties the ArrayList of properties to change it to
+     */
+    public void setProperties(ArrayList<Property> properties) {
+        this.properties = properties;
+    }
+
+
 }
 
```

Comment:
Good job of removing all the logic out of this entity, some of the JavaDoc needs to be written better, i.e. remove any time you say "basically" and any other unprofessional words.